articles/tutorials/qft-tutorial.md

```diff
@@ -0,0 +1,624 @@
+|1>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+|2>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+|3>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+|4>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+|5>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+|6>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+|7>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+```
+
+When called on the full-state simulator, `DumpMachine()` provides these mutliple representations of the quantum state's wavefunction. 
+The possible states of an $n$-qubit system can be represented by $2^n$ computational basis states, each with a corresponding complex coefficient (simply an amplitude and a phase).
+The computational basis states correspond to all the possible binary strings of length $n$---that is, all the possible combinations of qubit states $\ket{0}$ and $\ket{1}$, where each binary digit corresponds to an individual qubit.
+
+The first row provides a comment with the IDs of the corresponding qubits in their significant order.
+Qubit `2` being the "most significant" simply means that in the binary representation of basis state vector $\ket{i}$, the state of qubit `2` corresponds to the left-most digit. 
+For example, $\ket{6} = \ket{110}$ comprises qubits `2` and `1` both in $\ket{1}$ and qubit `0` in $\ket{0}$.
+
+
+The rest of the rows describe the probability amplitude of measuring the basis state vector $\ket{i}$ in both Cartesian and polar formats.
+In detail for the first row of our input state $\ket{000}$:
+* **`|0>:`** this row corresponds to the `0` computational basis state (given that our initial state post-allocation was $\ket{000}$, we would expect this to be the only state with probability amplitude at this point).
+* **`1.000000 +  0.000000 i`**: the probability amplitude in Cartesian format.
+* **` == `**: the `equal` sign seperates both equivalent representations.
+* **`********************`**: A graphical representation of the magnitude, the number of `*` is proportionate to the probability of measuring this state vector. 
+* **`[ 1.000000 ]`**: the numeric value of the magnitude
+* **`    ---`**: A graphical representation of the amplitude's phase.
+* **`[ 0.0000 rad ]`**: the numeric value of the phase (in radians).
+
+Both the magnitude and the phase are displayed with a graphical representation. The magnitude representation is straight-forward: it shows a bar of `*`, and the higher the probability, the larger the bar will be. For the phase, see the DumpMachine section [here](xref:microsoft.quantum.techniques.testing-and-debugging#dump-functions) for the possible symbol representations based on angle ranges.
```

Comment:
`s/straight-forward/straightforward/`

Did this patch already get merged, or is it cutting and pasting code from elsewhere in this repo? This exact sentence currently appears in testing-and-debugging.md.

Reply:
Yes, the majority of this explanation was simply lifted from there. Two reasons: 1) figured it better to simply explain the "need to know" here, rather than linking out to the specific section on the other page, and 2) that page will be getting a face lift in the coming weeks anyway